src/app/f2f/controllers/checkDetails.js

```diff
@@ -99,36 +98,35 @@ class CheckDetailsController extends DateController {
         case APP.PHOTO_ID_OPTIONS.NON_UK_PASSPORT: {
           idHasExpiryDate = req.form.values.idHasExpiryDate
           expiryDate = req.form.values.nonUKPassportExpiryDate;
-          country = req.form.values.nonUkPassportCountrySelector;
+          countryCode = req.form.values.nonUkPassportCountrySelector;
+          req.sessionModel.set("countryCode", countryCode);
+          req.sessionModel.set("country", res.locals.translate(`countries.${countryCode}`));
           break;
         }
         case APP.PHOTO_ID_OPTIONS.EU_PHOTOCARD_DL: {
           idHasExpiryDate = req.form.values.idHasExpiryDate
           expiryDate = req.form.values.euPhotocardDlExpiryDate;
-          country = req.form.values.euDrivingLicenceCountrySelector;
           address = req.form.values.euPhotocardDlAddressCheck
+          countryCode = req.form.values.euDrivingLicenceCountrySelector;
+          req.sessionModel.set("countryCode", countryCode);
+          req.sessionModel.set("country", res.locals.translate(`countries.${countryCode}`));
           break;
         }
         case APP.PHOTO_ID_OPTIONS.EEA_IDENTITY_CARD: {
           idHasExpiryDate = req.form.values.idHasExpiryDate
           expiryDate = req.form.values.eeaIdCardExpiryDate;
-          country = req.form.values.eeaIdentityCardCountrySelector;
           address = req.form.values.eeaIdentityCardAddressCheck;
+          countryCode = req.form.values.eeaIdentityCardCountrySelector;
+          req.sessionModel.set("countryCode", countryCode);
+          req.sessionModel.set("country", res.locals.translate(`countries.${countryCode}`))
```

Comment:
Do we need to set this is 3 places or can we just do `locals.countryTranslatedString = res.locals.translate(countries.${countryCode})`?

Reply:
The countryCode can't be set outside of those cases as it then overrides the UK ID options which gives rise to an undefined value for countryCode for UK passport, BRP and UK driving licence. The country and countryCode need to be set on the sessionModel for the purposes of testing, and the countryTranslatedString is just the display value for the check details screen

Reply:
I've removed countryCode as a variable as it wasn't needed, but have used the sessionModel variable country to set the countryTranslatedString as it's shorter - the reason for the country sessionModel variable is for the existing test expectations but apart from that it isn't used for anything